fvm/state/accounts.go

```diff
@@ -134,10 +134,6 @@ func (a *Accounts) GetPublicKeyCount(address flow.Address) (uint64, error) {
 		return 0, newLedgerGetError(keyPublicKeyCount, address, err)
 	}
 
-	if countBytes == nil {
-		return 0, nil
-	}
-
 	countInt := new(big.Int).SetBytes(countBytes)
```

Comment:
I was also wondering why we need to use `big.Int`. Couldn't we just test that `len(countBytes) > 8` instead of `!countInt.IsUint64()` ?

Reply:
I don't know if we _need_ to use it per se, but it does have the same effect, and visually is more clear, i.e. `!countInt.IsUint64()` vs `len(countBytes) > 8`

Reply:
I meant that using `big.Int` just for the check comes with a slight performance overhead that we could avoid, although I agree we lose a bit in code readability.